### PR TITLE
Jamstack requirements

### PR DESCRIPTION
**Pull request by Jonic**
Couple of things that cropped up during development of the new home page:

* Reduced motion helper
* `_accessibility.scss` base file that disables `animation` and `transition` if user's system requests reduced motion
* New "space mono" monospace font family
* Guidance on inclusion of fonts in README
* browserslist config file - see https://github.com/browserslist/browserslist#config-file

### CODE REVIEW

File: src/styles/base/_accessibility.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@media screen and (prefers-reduced-motion: reduce) {
```
Comment (aldreth) [inline src/styles/base/_accessibility.scss]: Interesting!

Comment (Jonic) [reply src/styles/base/_accessibility.scss]: Yeah, have a look at: https://css-tricks.com/introduction-reduced-motion-media-query/ :)

**Comment by Jonic** [comment]
@aldreth Code Climate was sad because I've now got Prettier formatting my code. Disabled scss-lint on Code Climate because it should be switched to stylelint. When I eventually get chance to spend time with this I'll sort that out :)

File: src/styles/base/_accessibility.scss (hunk 1 above)
Comment (magdalenajadach) [reply src/styles/base/_accessibility.scss]: It's a great read! Thanks @Jonic!

**Comment by Jonic** [comment]
@aldreth Accidentally dismissed your review! Can you review again in the morning please? TA! :)

**Comment by Jonic** [comment]
@aldreth I'm restoring this branch to add a couple of things to it. I forgot the version bump, and the change log. I should add those to a PR template, so I'm going to do that too!

**Pull request merged**